src/Grid/EventButton.vala

```diff
@@ -55,17 +57,26 @@ public class Maya.View.EventButton : Gtk.Revealer {
                 Gtk.Menu menu = new Gtk.Menu ();
                 menu.attach_to_widget (this, null);
 
-                var edit_item = new Gtk.MenuItem.with_label (_("Edit…"));
-                var remove_item = new Gtk.MenuItem.with_label (_("Remove"));
+                bool sensitive = src.writable == true && Model.CalendarModel.get_default ().calclient_is_readonly (src) == false;
 
-                if (src.writable != true && Model.CalendarModel.get_default ().calclient_is_readonly (src) != false) {
-                    edit_item.sensitive = false;
-                    remove_item.sensitive = false;
+                Gtk.MenuItem edit_item = new Gtk.MenuItem.with_label (_("Edit…"));
+                edit_item.activate.connect (() => { edition_request (); });
```

Comment:
This could be `edit_item.activate.connect (edition_request);`

Reply:
edition_request is a signal, and Vala does not seem to like connecting it in this way, changing the others to this form was fine though.